Modules/CraftQueue/CraftQueue.lua

```diff
@@ -358,7 +371,9 @@ function CraftSim.CRAFTQ:ImportRecipeScan()
                             end
                         end
                     end
-
+                    if restockAmount == nil then
+                        restockAmount = CraftSim.DB.OPTIONS:Get("CRAFTQUEUE_GENERAL_RESTOCK_RESTOCK_AMOUNT")
+                    end
                     if restockAmount > 0 then
                         if recipeData.cooldownData.isCooldownRecipe then

```

Comment:
If you want you can try putting the logic to get the restockAmount into a function to clear the clutter a bit. But I can also do a cleanup some day after ^^